### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by gityerhubhere** [dalibo/pgbadger#241]
If a column isn't all lowercase (which is frequently the case), pgBadger incorrectly turns it into lowercase, such as "URL" => "url", breaking things, and requiring me to manually edit the "normalized" queries for them to be possible to run.

**Comment by darold** [comment]
Yes that's a know issue with pgbadger. To be able to produce unique normalized queries and to be able to compute statistics on unique queries pgbadger turns the query in lowercase.  Trying to preserve case sensitivity in those objects names would cost too many CPU cycles for not so much cases. At least you are the first user that is complaining about that. If you can don't use case sensitivity, but maybe you can not change the application easily, sorry for that.

**Comment by gityerhubhere** [comment]
I specifically do not use lowercase because I want to name them correctly. "url" is a noise or something, whereas "URL" means something. I frankly don't understand what's so wasteful about doing this check. Or why it's wasteful at all. :/

**Comment by darold** [comment]
Well actually we just perform a full lc($query) which is something that have a cost but less than having to detect words between double quotes and and keep them with there case sensitivity. This reported to millions of lines can cost a lot of time. I don't understand your explanation about url vs URL, but we can have the same problem with queries ported from others RDBMS.

**Comment by darold** [comment]
Here is a screenshot of what you should obtain with "case sensitive" queries:

![case_sensitivity](https://cloud.githubusercontent.com/assets/538862/8465135/d78467b6-2047-11e5-9fe4-ae14998bf267.png)

as you can see normalized queries are turned to lower case but you still have untouched queries in example list. So honestly I don't really understand what's wrong. You may want to use a sample to try to execute the query and not the normalized one.
